sklearn/model_selection/_search.py

```diff
@@ -355,7 +355,7 @@ def _check_refit(search_cv, attr):
         )
 
 
-def _estimator_has(attr):
+def _search_estimator_has(attr):
```

Comment:
The `_estimator_has` method in this file has a custom behavior of using `_check_refit(self, attr)` method inside.I updated the method name to make it clearer.